src/fermilib/utils/_sparse_tools.py

```diff
@@ -327,6 +328,45 @@ def expectation(sparse_operator, state):
     return expectation
 
 
+def expectation_product_state(operator, product_state):
+    """Compute expectation value of an operator with a product state.
+
+    Args:
+        operator: Qubit or FermionOperator to evaluate expectation value of.
+                  If operator is a FermionOperator, it must be normal-ordered.
+        product_state (scipy.sparse vector): normalized pure state.
```

Comment:
Normalized pure state or normalized product state? Do you really mean a product state or do you mean a computational basis state?